package.json

```diff
@@ -8,136 +8,71 @@
-    "@storybook/addons": "^3.4.10",
-    "@storybook/react": "^3.4.10",
-    "@storybook/storybook-deployer": "^2.1.0",
-    "babel-core": "6.26.0",
-    "babel-eslint": "7.2.3",
-    "babel-jest": "20.0.3",
-    "babel-loader": "7.1.2",
-    "babel-preset-react-app": "^3.1.1",
-    "babel-runtime": "6.26.0",
-    "chalk": "1.1.3",
-    "coveralls": "^3.0.2",
-    "css-loader": "0.28.7",
-    "dotenv": "4.0.0",
-    "dotenv-expand": "4.2.0",
-    "enzyme": "^3.5.1",
-    "enzyme-adapter-react-16": "^1.3.1",
-    "enzyme-to-json": "^3.3.4",
-    "eslint": "^4.19.1",
-    "eslint-config-prettier": "^2.9.0",
-    "eslint-config-standard": "^10.2.1",
-    "eslint-config-standard-jsx": "^4.0.2",
-    "eslint-config-standard-react": "^5.0.0",
-    "eslint-import-resolver-webpack": "^0.9.0",
-    "eslint-loader": "^2.1.0",
-    "eslint-plugin-import": "^2.10.0",
-    "eslint-plugin-jest": "^21.15.0",
-    "eslint-plugin-jsx-a11y": "^6.0.3",
-    "eslint-plugin-node": "^5.2.1",
-    "eslint-plugin-patternfly-react": "*",
-    "eslint-plugin-prettier": "^2.4.0",
-    "eslint-plugin-promise": "^3.7.0",
-    "eslint-plugin-react": "^7.7.0",
-    "eslint-plugin-standard": "^3.0.1",
-    "extract-text-webpack-plugin": "3.0.2",
-    "file-loader": "1.1.5",
-    "fs-extra": "3.0.1",
-    "html-webpack-plugin": "2.29.0",
-    "jest": "20.0.4",
-    "lodash": "^4.17.11",
-    "object-assign": "4.1.1",
-    "patternfly-react": "^2.14.0",
-    "postcss-flexbugs-fixes": "3.2.0",
-    "postcss-loader": "2.0.8",
-    "prettier": "^1.14.2",
-    "prettier-eslint": "^8.8.1",
-    "promise": "8.0.1",
-    "prop-types": "^15.6.2",
-    "raf": "3.4.0",
-    "react": "^16.4.2",
-    "react-dev-utils": "^5.0.1",
-    "react-dom": "^16.4.2",
-    "resolve": "1.6.0",
-    "style-loader": "0.19.0",
-    "stylelint": "^8.4.0",
-    "stylelint-config-standard": "^18.2.0",
-    "sw-precache-webpack-plugin": "0.11.4",
-    "url-loader": "0.6.2",
-    "webpack": "3.8.1",
-    "webpack-dev-server": "2.9.4",
-    "webpack-manifest-plugin": "1.3.2",
-    "whatwg-fetch": "2.0.3"
+  "files": [
+    "/dist"
+  ],
+  "scripts": {
+    "lint": "yarn lint-js && yarn lint-css",
+    "lint-js": "yarn lint-js-browser && yarn lint-js-node",
+    "lint-js-browser": "eslint src --config config/eslint.browser.js",
+    "lint-js-node": "eslint config tools --config config/eslint.node.js",
+    "lint-css": "stylelint 'src/**/*.css' --config config/stylelint.config.js",
+    "prevalidate": "yarn lint",
+    "validate": "node tools/runValidations.js",
+    "pretest": "yarn validate",
+    "test": "rimraf coverage && jest --config config/jest.config.js",
+    "test-watch": "yarn test --watch",
+    "prebuild": "yarn test --ci",
+    "build": "rimraf dist && yarn build-js && yarn build-css",
+    "build-js": "babel src --config-file ./config/babel.config.js --out-dir dist --ignore '**/*.test.js,**/setupTest.js'",
+    "build-css": "cd src && find . -name '*.css' -exec cp {} ../dist/{} \\;",
```

Comment:
How are `.css` imported by consuming application?

Considering project context, I think we should not import styling from components.
Use of sass and hierarchical imports with single entry-point seems to be better approach.

Reply:
> Use of sass and hierarchical imports with single entry-point seems to be better approach.

This is what `patternfly-react` does too, I'm OK with that. We can have a single (s)css entry point to be imported by the consuming application, in order to pull in all the necessary styling.

In our components, we can simply refer to CSS class names via [`className` prop](https://reactjs.org/docs/faq-styling.html).

Adding Sass to the project is on my task list.